src/components/WorkflowEditor/AddTaskHaley/subcomponents/AddTask1.js

```diff
@@ -58,17 +58,19 @@ class AddTask1 extends Component {
         return (
         <>
             <p>Next, enter some basic details about the task:</p>
-                <div>
-                    <label htmlFor="titleInput">
-                        Title:<input type="text" id="titleInput" onChange={this.handleTitleInput}/>
-                    </label>
-                </div>
+            <form className="form">
+                <li>
```

Comment:
does this need to be within a <ul> tag?

Reply:
form is just a classname, like how i use button, its dumb i know, but lets you style the class and not the tag